doc/en/components/grids/_shared/column-hiding.md

```diff
@@ -130,11 +179,103 @@ For this purpose all we have to do is set both the `GridToolbarActions` and the
     </IgrGridToolbar>
 </IgrGrid>
 ```
+<!-- ComponentEnd: Grid, TreeGrid -->
+
+<!-- Angular -->
+<!-- ComponentStart: HierarchicalGrid -->
+```html
+<igx-hierarchical-grid [data]="localdata">
+    <igx-grid-toolbar>
+        <igx-grid-toolbar-actions>
+            <igx-grid-toolbar-hiding></igx-grid-toolbar-hiding>
+        </igx-grid-toolbar-actions>
+    </igx-grid-toolbar>
+
+    <igx-row-island>
```

Comment:
This will not work even when the toolbar get's fixed. As a template, the assigning happens in the ts  file only:
this._bind = () => {
rowIsland1.toolbarTemplate = this.webHierarchicalGridHidingTemplate;
}
so we can add such snippet later on, but for now please remove the html part for the row island.